modules/gear/loadouts/basic.sqf

```diff
@@ -5,8 +5,10 @@
 /* Item Slots */
 ADD_WEAPON("ItemMap");
 ADD_WEAPON("ItemWatch");
+ADD_WEAPON("ItemGPS");

```

Comment:
I dont think that everyone needs GPS. We already have moving map and I dont like the idea of everyone having GPS as a base thing

Reply:
Then maybe we can remove GPS entirely? I think it's in the "crew" and SL/TL sqf's.

Reply:
That's what I'm getting at. I would rather it be leader specific rather than everyone getting one. It allows a more zoomed in view as well as a more immediate update than the moving map so it is a "special item".